fvtest/jitbuildertest/CMakeLists.txt

```
@@ -44,7 +44,7 @@ if(OMR_HOST_ARCH STREQUAL "x86")
 endif()
 
 if(NOT OMR_HOST_ARCH STREQUAL "ppc")
-	target_sources(jitbuildertest PUBLIC UnsignedDivRemTest.cpp)
+        target_sources(jitbuildertest PRIVATE UnsignedDivRemTest.cpp)
```

I think inconsistent use of tabs and spaces within a file can make the file unnecessarily difficult to read. Please change spaces back to a tab so it is consistent with the rest of the file.

```suggestion
	target_sources(jitbuildertest PRIVATE UnsignedDivRemTest.cpp)
```